packages/react-components/react-popover/etc/react-popover.api.md

```diff
@@ -60,7 +60,7 @@ export type PopoverProps = Pick<PortalProps, 'mountNode'> & {
 };
 
 // @public (undocumented)
-export const PopoverProvider: Provider<PopoverContextValue> & FC<ProviderProps<PopoverContextValue>>;
+export const PopoverProvider: Provider<PopoverContextValue | undefined> & FC<ProviderProps<PopoverContextValue | undefined>>;
```

Comment:
Unrelated: is it okay that we export this at all?

Reply:
I don't understand, why shouldn't we export it ?

Reply:
It's not documented publicly so I decided to ask 🐱